org.adempiere.base/src/org/adempiere/base/AnnotationBasedColumnCalloutFactory.java

```diff
@@ -161,17 +161,59 @@ public void activate(ComponentContext context) throws ClassNotFoundException {
 				.acceptPackagesNonRecursive(getPackages());
 
 		ScanResultProcessor scanResultProcessor = scanResult -> {
+			/**
+			 * NOTE1: callout implement IColumnCallout use annotation Callout for 2 case
+			 * case 1:single Callout
+			 * case 2:multi Callout (by that it treats as Callouts)
+			 * 
+			 * for case 1:
+			 *   scanResult.getClassesWithAnnotation(Callouts.class) return empty list
+			 *   scanResult.getClassesWithAnnotation(Callout.class) return non-empty list
+			 * for case 2:
+			 *   scanResult.getClassesWithAnnotation(Callouts.class) return non-empty list
+			 *   scanResult.getClassesWithAnnotation(Callout.class) return empty list
+			 *   
+			 *  but CalloutInfoWindow is special case (still don't know why)
+			 *  it return non-empty list for both called so need to check class processed to avoid duplicate registration callout
+			 */
 			List<String> processed = new ArrayList<String>();
 		    for (ClassInfo classInfo : scanResult.getClassesWithAnnotation(Callouts.class)) {
 		    	if (classInfo.isAbstract())
 		    		continue;
-		        String className = classInfo.getName();		        
-		        AnnotationInfoList annotationInfos = classInfo.getAnnotationInfoRepeatable(Callout.class);
-		        for(AnnotationInfo annotationInfo : annotationInfos) {
-			        processAnnotation(className, annotationInfo);
+		        String className = classInfo.getName();
+		        /**
+		         * NOTE2: continue of special case for CalloutInfoWindow
+		         * normal case we get bellow result:
+		         * 		classInfo.getAnnotationInfoRepeatable(Callouts.class) return list (1 item) AnnotationInfo of Callouts
+		         *      classInfo.getAnnotationInfoRepeatable(Callout.class) return empty list
+		         *		classInfo.getAnnotationInfo(Callouts.class) AnnotationInfo of Callouts
+		         *		classInfo.getAnnotationInfo(Callout.class) return null
+		         * "CalloutInfoWindow" get bellow result
+		         * 		classInfo.getAnnotationInfoRepeatable(Callouts.class) return empty list
+		         *      classInfo.getAnnotationInfoRepeatable(Callout.class) return list (2 item) AnnotationInfo of Callout
+		         *		classInfo.getAnnotationInfo(Callouts.class) return null
+		         *		classInfo.getAnnotationInfo(Callout.class) return latest AnnotationInfo of Callout
+		         *
+		         * so use classInfo.getAnnotationInfo() to get list of AnnotationInfo of Callout
+		         */
```

Comment:
This can be simplify to scenario 1 and 2, that's we should have "Scenario 1:" and "Scenario 2 (CalloutInfoWindow):"